modules/ROOT/pages/lb-create-arm.adoc

```diff
@@ -56,28 +56,32 @@ Redirects the request to the same URL using the HTTPS protocol.
 
 . Add a certificate.
 
-.. Click *Add Certificate*
-.. Add both a public and a private key.
-.. (Optional) Select *Support TLS 1.0*.
-.. (Optional) Add a client certificate.
+.. Click *Add certificate*
++
+.The arrow shows the *Add certificate* option in the *Create Load Balancer* page.
+image::lb-cert-upload-create.png[Add certificate option in the Create Load Balancer page]
+.. On the *Create Load Balancer | Add certificate* page, select *Choose File* to upload both public key and private key files.
+.. If you want to add a client certificate, click *Choose File* to upload the file.
 +
 The client certificate is a trusted CA certificate used to verify client certificates.
 
-.. (Optional) Add URL mapping rules.
+.. If you want to add URL mapping rules, click the expand icon to display the options:
```

Comment:
I'm assuming that "the expand icon" is "+", which others mostly refer to as "the plus sign".
I like yours much better.
But I wonder if we could replace both with "click *+*"?

Reply:
It's not a plus - it's the > widget. I guess I should say "click >".

Reply:
Just looked it up - it's called the "disclosure widget".